python/stempy/pipeline/__init__.py

```diff
@@ -8,12 +8,36 @@ def execute(self, **kwargs):
         Execute pipeline
         """
 
+def _ensure_parameters(f):
+    if not hasattr(f, 'PARAMETERS'):
+        setattr(f, 'PARAMETERS', {})
+
 def pipeline(name=None, description=None):
 
     def _decorator(func):
         func.NAME = name
         func.DESCRIPTION = description
-
+        _ensure_parameters(func)
```

Comment:
Do we need this one? We are already calling it in the decorator.

Reply:
If there is ever a pipeline with no parameters, this ensures that there is an empty `PARAMETERS` attribute on the function

Reply:
Ok, lets remove it from the other decorator then.